src/displayapp/screens/WatchFaceTerminal.cpp

```diff
@@ -0,0 +1,264 @@
+  lv_obj_align(bleIcon, batteryPlug, LV_ALIGN_OUT_LEFT_MID, -5, 0);
+
+  notificationState = notificatioManager.AreNewNotificationsAvailable();
+  if(notificationState.IsUpdated()) {
+    if(notificationState.Get() == true)
+      lv_label_set_text(notificationIcon, NotificationIcon::GetIcon(true));
+    else
+      lv_label_set_text(notificationIcon, NotificationIcon::GetIcon(false));
+  }
+
+  currentDateTime = dateTimeController.CurrentDateTime();
+
+  if(currentDateTime.IsUpdated()) {
+    auto newDateTime = currentDateTime.Get();
+
+    auto dp = date::floor<date::days>(newDateTime);
+    auto time = date::make_time(newDateTime-dp);
+    auto yearMonthDay = date::year_month_day(dp);
+
+    auto year = (int)yearMonthDay.year();
+    auto month = static_cast<Pinetime::Controllers::DateTime::Months>((unsigned)yearMonthDay.month());
+    auto day = (unsigned)yearMonthDay.day();
+    auto dayOfWeek = static_cast<Pinetime::Controllers::DateTime::Days>(date::weekday(yearMonthDay).iso_encoding());
+
+    auto hour = time.hours().count();
+    auto minute = time.minutes().count();
+	  auto second = time.seconds().count();
+
+    char minutesChar[6];
+    sprintf(minutesChar, "%02d", static_cast<int>(minute));
+
+    char hoursChar[8];
+
+    char ampmChar[3];
+	    if (settingsController.GetClockType() == Controllers::Settings::ClockType::H24) {
+        sprintf(hoursChar, "%02d", hour);
+      } else {
+        if (hour == 0 && hour != 12) {
+          hour = 12;
+          sprintf(ampmChar, "AM");
+        } else if (hour == 12 && hour != 0) {
+          hour = 12;
+          sprintf(ampmChar, "PM");
+        } else if (hour < 12 && hour != 0) {
+          sprintf(ampmChar, "AM");
+        } else if (hour > 12 && hour != 0) {
+          hour = hour - 12;
+          sprintf(ampmChar, "PM");
+        }
+        sprintf(hoursChar, "%02d", hour);
+    }
+
+	char secondsChar[5];
+    sprintf(secondsChar, "%02d", static_cast<int>(second));
+
+	auto batteryValue = static_cast<uint8_t>(batteryController.PercentRemaining());
+/*
+	char batteryGauge[12];
+    if (batteryValue > 75){
+      sprintf(batteryGauge, "[++++]");  //100-75
+    }
+    else if (batteryValue > 50){
+      sprintf(batteryGauge, "[+++.]");  //75-50
+    }
+    else if (batteryValue > 25){
+      sprintf(batteryGauge, "[++..]");  //50-25
+    }
+    else if (batteryValue > 10){
+      sprintf(batteryGauge, "[+....]"); //25-10
+    }
+    else if (batteryValue > 0){
+      sprintf(batteryGauge, "[!!!!!]"); //10-0
+    }
+*/
+    char battStr[24];
+    sprintf(battStr, "[BATT]#387b54 %d%\%#", batteryValue);
+    lv_label_set_text(batteryPercent, battStr);
+
+    if(hoursChar[0] != displayedChar[0] ||
+       hoursChar[1] != displayedChar[1] ||
+       minutesChar[0] != displayedChar[2] ||
+       minutesChar[1] != displayedChar[3] ||
+       secondsChar[0] != displayedChar[4] ||
+       secondsChar[1] != displayedChar[5])
```

Comment:
move the opening bracket at the end of the `if` statement (and delete the one in the next line, but don't know how to to that with the GitHub suggest-changes feature)
```suggestion
       secondsChar[1] != displayedChar[5]) {
```